test/MVCFramework45.FunctionalTests/FunctionalTest/EntityFrameworkTelemetryTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace MVCFramework45.FunctionalTests.FunctionalTest
+{
+    using System.Linq;
+    using System.Net.Http;
+    using FunctionalTestUtils;
+    using Microsoft.ApplicationInsights.DataContracts;
+    using Xunit;
+
+    public class EntityFrameworkTelemetryTests : TelemetryTestsBase
+    {
+        private const string assemblyName = "MVCFramework45.FunctionalTests";
+
+        [Fact]
+        public void TestEntityFrameworkTelemetryItemsReceived()
+        {
+            InProcessServer server;
+            using (server = new InProcessServer(assemblyName, InProcessServer.UseApplicationInsights))
+            {
+                using (var httpClient = new HttpClient())
+                {
+                    var task = httpClient.GetAsync(server.BaseHost + "/Home/Contact");
+                    task.Wait(TestTimeoutMs);
+                }
+            }
+            var telemetries = server.BackChannel.Buffer.OfType<DependencyTelemetry>()
```

Comment:
What is the type here?  I'm guessing it might be "DependencyTelemetry[]" but I'm not sure, please replace var with the known type.